File: lista4/numero_primo.py
N = int(input())
while N > 0:
    X = int(input())
    somatorio = 0
    for i in range(1,X+1):
        if X%i==0:
            somatorio+=1
    if somatorio==2:
        print(f'{X} eh primo')
    else: 
        print(f'{X} nao eh primo')
    N-=1

File: lista4/numeroPrimo.js
let N = parseInt(prompt())
while(N > 0){
    const X = parseInt(prompt())
    let somatorio = 0
    for(let i = 1; i < X+1; i++ ){
        if(X%i==0){
            somatorio = somatorio + 1
        }
    }
    if(somatorio==2){
        console.log(`${X} eh primo`)
    } else{
        console.log(`${X} nao eh primo`)
    }
    N = N -1
}